### DISCUSSION TITLE
Declaração de isenção das retenções para exa

### DISCUSSION THREAD

**Discussion by emillyhorta** [apyb/comunidade#147] [Pergunta]
Oi, a EXA nos mandou a seguinte mensagem : Após análise fiscal da NFS-e 364 Associação Python, em regra geral o código de atividade de treinamento, está sujeito a retenção de IRRF e PCC, gentileza solicito o envio da declaração que possa ser enviada informando o motivo da dispensa das retenções.

@cecivieira

**Comment by cecivieira** [comment]
redirecionada para #134
